core/domain/search_services_test.py

```diff
@@ -119,17 +119,17 @@ def test_search_ranks_cannot_be_negative(self):
     def test_search_explorations(self):
         expected_query_string = 'a query string'
         expected_cursor = 'cursor'
-        expected_limit = 30
+        expected_size = 30
         expected_result_cursor = 'rcursor'
         doc_ids = ['id1', 'id2']
 
         def mock_search(
-                query_string, index, cursor=None, size=20,
+                query_string, index, offset=None, size=20,
                 ids_only=False, retries=3):
             self.assertEqual(query_string, expected_query_string)
             self.assertEqual(index, search_services.SEARCH_INDEX_EXPLORATIONS)
-            self.assertEqual(cursor, expected_cursor)
-            self.assertEqual(size, expected_limit)
+            self.assertEqual(offset, expected_cursor)
```

Comment:
Same thing, here and elsewhere. Please do not compare an offset and a cursor, or a size and a limit. This doesn't make sense since the two concepts are not synonyms (and even if they were it would be better to collapse them to one concept anyway).

Please also make sure that you _are_ actually dealing with offsets throughout before making the changes.

Reply:
Done in https://github.com/oppia/oppia/pull/11295/commits/93a123cc7869c29b5a8b9e86d62b7fecb13f7a5a.